scripts/download-helper.js

```diff
@@ -0,0 +1,101 @@
+const fs = require('fs');
+const https = require('https');
+const zlib = require('zlib');
+const path = require('path');
+const tar = require('tar');
+const sha384 = require('sha384');
+
+// TODO: Change URL from fork to @runk's repo (and master branch)
+const link = file => `https://raw.githubusercontent.com/GitSquared/node-geolite2/new-eula-redistribution/redist/${file}`;
+
+const editions = [
+  { name: 'GeoLite2-ASN' },
+  { name: 'GeoLite2-City' },
+  { name: 'GeoLite2-Country' }
+];
+
+editions.forEach((edition, index) => {
+  editions[index].dbURL = link(edition.name+'.tar.gz');
+  editions[index].checksumURL = link(edition.name+'.mmdb.sha384');
+});
+
+function fetchChecksums() {
+  let downloads = [];
+
+  editions.forEach(edition => {
+    downloads.push(new Promise(resolve => {
+      https.get(edition.checksumURL, res => {
+        let checksum = '';
+        res.on('data', chunk => {
+          checksum = checksum+chunk.toString();
+        });
+        res.on('end', () => {
+          checksum = checksum.trim();
+          if (!res.complete || checksum.length !== 96) throw new Error(`Could not fetch checksum for ${edition.name}\n\nReceived:\n${checksum}`);
+          edition.checksum = checksum;
```

Comment:
I'm not a big fan of monkey-patching in real time. Any way we can avoid it?

Reply:
I'm not sure I understand your comment here - what monkey-patching are you referring to?

Reply:
You're extending items in `editions` list asynchronously

Reply:
Got it (I think). Will fix this to only change `editions` if all checksums have successfully been retrieved.

Reply:
7d2263c